src/etc/build_rust.py

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/python
```

Comment:
This is also positively something that will never work on systems where `python == python3 == python3.*`.